openwisp_monitoring/device/templates/admin/config/device/change_form.html

```diff
@@ -620,6 +622,12 @@ <h2>{% trans 'Charts' %}</h2>
       location.href = baseUrl + time + '&csv=1';
     })
   });
+  let elements = document.getElementsByClassName("bridgeMember");
+  for(let i=0;i<elements.length;i++) {
+      elements[i].addEventListener("click", ()=>{
+          window.scrollTo(0, 1800+500*i);
```

Comment:
What is this? I believe you should scroll to a specific ID, why these numbers? Are you sure it's gonna work? Looks fragile to me.

Also here the code style is not consistent, look at quote style and spacing within the loop condition.

Reply:
I agree, it feels like a scary piece of code for two reasons:
1. I don't understand it completely, why is it working?
2. I think if something changes, it'll break.

Might be worth thinking of another solution indeed! 🤔

Reply:
Ideally, it'll be something like:

```
{% for element in interface.bridge_members %}
    <span class="bridge-member" onclick="scrollToInterface(<arguement-identifing-target>)">{{ element }}</span>,
{% endfor %}

function scrollToInterface(target) {
// your-logic-here
}
```

This is still better, because there is no external loop in which scrolling part depends.
More maintainable.

> I believe you should scroll to a specific ID,

This idea sounds much less scary and reliable, can we try it out instead of those numbers? 😄